web/geosearch/datapunt_geosearch/config.yaml

```diff
@@ -0,0 +1,30 @@
+jwks_url: ""
+
+jwks: >
+  {
+    "keys": [
+        {
+          "kty": "EC",
+          "key_ops": [
+            "verify",
+            "sign"
+          ],
+          "kid": "2aedafba-8170-4064-b704-ce92b7c89cc6",
+          "crv": "P-256",
+          "x": "6r8PYwqfZbq_QzoMA4tzJJsYUIIXdeyPA27qTgEJCDw=",
+          "y": "Cf2clfAfFuuCB06NMfIat9ultkMyrMQO9Hd2H7O9ZVE=",
+          "d": "N1vu0UQUp0vLfaNeM0EDbl4quvvL6m_ltjoAXXzkI3U="
+      }
+    ]
+  }
+  
+jwks_allowed_signing_algorithms:
```

Comment:
In authorization_django zijn een aantal hashing algoritmes (de asymetrische) zojuist verwijderd ivm security. Wellicht hier ook doen?
https://github.com/Amsterdam/authorization_django/commit/7d9533e9e3dba496d96a38c4246685ea7ff3a8b5

Reply:
Thnx. Aangepasst.